src/components/molecules/Help/Help.tsx

```diff
@@ -9,32 +9,61 @@ import Card from '../../organisms/Card';
 import { buttonStyle } from '../../atoms/Button/Button';
```

Comment:
As a general comment - I'd add some more vertical spacing between the various sections of the code.

Reply:
I agree, makes it more readable.